webapp/pages/moderation/index.vue

```diff
@@ -82,6 +82,14 @@
           {{ scope.row.submitter.name }}
         </nuxt-link>
       </template>
+      <!-- createdAt -->
+      <template slot="createdAt" slot-scope="scope">
```

Comment:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-10-14T21:06:21Z" title="Monday, October 14th 2019, 11:06:21 pm +02:00">Oct 14, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -82,6 +82,14 @@
           {{ scope.row.submitter.name }}
         </nuxt-link>
       </template>
+      <!-- createdAt -->
+      <template slot="createdAt" slot-scope="scope">
```


Seems to be a deprecated API: https://vuejs.org/v2/guide/components-slots.html#Scoped-Slots-with-the-slot-scope-Attribute